exporter/googlecloudexporter/config_test.go

```diff
@@ -29,7 +28,6 @@ func TestCreateDefaultConfig(t *testing.T) {
 	googleCfg, ok := cfg.(*Config)
 	require.True(t, ok)
 
-	require.Equal(t, component.NewID(typeStr), googleCfg.ID())
```

Comment:
i believe you that this is fine, but can you explain why we don't need this test now?

Reply:
The ID is no longer on the config but rather on the settings.